projects/samples/devices/controllers/lidar/lidar.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Simulation of a lidar.
+"""
+
+from controller import Robot
+import random
+
+
+class Controller(Robot):
+    def __init__(self):
+        super(Controller, self).__init__()
+        self.timeStep = 32
+        self.lidar = self.getDevice('lidar')
+        self.lidar.enable(self.timeStep)
+        self.lidar.enablePointCloud()
+        self.us0 = self.getDevice('us0')
+        self.us0.enable(self.timeStep)
+        self.us1 = self.getDevice('us1')
+        self.us1.enable(self.timeStep)
+        self.left_motor = self.getDevice('left wheel motor')
+        self.right_motor = self.getDevice('right wheel motor')
+        self.left_motor.setPosition(float('inf'))
+        self.right_motor.setPosition(float('inf'))
+        self.left_motor.setVelocity(0)
+        self.right_motor.setVelocity(0)
+
+    def run(self):
+        # empirical coefficients for collision avoidance
+        coefficients = [[12.0, -6.0], [-10.0, 8.0]]
+        base_speed = 6.0
+        speed = [0, 0]
+        us_value = [0, 0]
+        n = self.lidar.getNumberOfPoints()
+        while self.step(self.timeStep) != -1:
+            value = self.lidar.getRangeImage()
+            i = random.randrange(n)
+            print(f'range[{i}] = {value[i]}')
+            points = self.lidar.getPointCloud()
+            if points:
+                print(f'point[{i}] = {points[i].x} {points[i].y} {points[i].z} {points[i].layer} {points[i].time}')
```

Comment:
Was it for debugging or is it on purpose?
Because it is not in the C code and it is flooding the terminal.